BitmovinYoSpaceModule/Classes/BitmovinYospacePlayer.swift

```diff
@@ -806,16 +806,7 @@ public extension BitmovinYospacePlayer {
         let policy = yospacePlayerPolicy ?? YospacePlayerPolicy(bitmovinYospacePlayerPolicy: DefaultBitmovinYospacePlayerPolicy(self))
         stream.setPlaybackPolicyHandler(policy)
 
-        switch stream.sessionResult {
-        case .notInitialised:
-            BitLog.e("Not initialized url:\(stream.playbackUrl ?? "none") itemId:\(stream.identifier ?? "none")")
-            if let sourceConfiguration = sourceConfig, yospaceSourceConfig?.retryExcludingYospace == true {
-                BitLog.w("Attempting to playback the stream url without Yospace")
-                emitYoSpaceWarning(YospaceWarningEvent(errorCode: .notIntialised, message: "Yospace not initialized"), player: self)
-                load(sourceConfig: sourceConfiguration)
-            } else {
-                emitYoSpaceError(YospaceErrorEvent(errorCode: .notIntialised, message: "Yospace not initialized"), player: self)
-            }
+        switch stream.sessionState {
```

Comment:
The update had some breaking changes

Reply:
Hm 🤔 what do you mean with the point you there? We are using the `YOSessionState` enum?

![Screenshot 2025-04-14 at 08 46 05](https://github.com/user-attachments/assets/e5940138-14f2-4463-981c-c5169a07d8d7)

Reply:
Hm, when I use Xcode to jump to the definition of e.g. `.noAnalytics`, it leads me to https://github.com/bitmovin/bitmovin-player-ios-integrations-yospace/blob/d9e97efd6c782ee5bf36158c1f419818a1a856fb/BitmovinYoSpaceModule/Classes/YospaceEventCode.swift#L8

Maybe Xcode is fooling me 😄  Happy to sync offline if that's easier

Reply:
Looks like your Xcode is still in April fool mode (well, it's in this mode most of the year 😅)

Everything seems to be fine on my end 🙂 
![Screenshot 2025-04-14 at 09 22 20](https://github.com/user-attachments/assets/6becebf6-ceed-44f9-a26c-531b8fbb8c40)

Reply:
This makes more sense. Thanks for clarifying 🙂